.github/workflows/lint-test.yml

```diff
@@ -29,8 +29,7 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
-      - run: yarn --frozen-lockfile
-      - run: yarn allow-scripts
+      - run: yarn --immutable --immutable-cache
```

Comment:
Hmm... maybe we don't want `--immutable-cache` after all, since it will fail if the cache folder doesn't exist (see CI). @Gudahtt What do you think?

Reply:
The `--immutable-cache` flag was meant to be used in the post-prepare jobs, where it was guaranteed the cache would be populated. Agreed that we shouldn't use it here.

Reply:
Agh, my bad, that's right. Sorry @adonesky1 we don't want this after all, so this should just be `yarn --immutable` 😅

Reply:
done [here](https://github.com/MetaMask/eth-simple-keyring/pull/119/commits/4ee66373cd649465f68e4f7095c6b259ef137a7f)